project/.travis.yml.twig

```diff
@@ -70,6 +70,11 @@ matrix:
     - php: nightly
     - env: SYMFONY_DEPRECATIONS_HELPER=0
 {% for package_name,package_versions in versions %}
+{% if package_name|lower == 'sonata_block' %}
```

Comment:
The `projects.yml` file should be a simple configuration file.

I'm really uncomfortable with this hack, can't we just adjust the configuration file (without comment but remove if justified)?